File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/CinemaHallDao.java
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api;

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface;
import ru.kami.gis.afisha.schedule.core.domain.CinemaHallEntity;

import java.util.List;
import java.util.Optional;

/**
 * @author Daniil.Makarov
 */
public interface CinemaHallDao extends JdbcDaoInterface {
    Optional<List<CinemaHallEntity>> findAllByIdCinema(Long id);
}

File: afisha-schedule/afisha-schedule-core/src/main/java/ru/kami/gis/afisha/schedule/core/repository/jdbc/api/CinemaHallDao.kt
package ru.kami.gis.afisha.schedule.core.repository.jdbc.api

import ru.kami.gis.afisha.schedule.api.repository.JdbcDaoInterface
import ru.kami.gis.afisha.schedule.core.domain.CinemaHallEntity

/**
 * @author Daniil.Makarov
 */
interface CinemaHallDao : JdbcDaoInterface {
    fun findAllByIdCinema(id: Long): List<CinemaHallEntity?>
}